### DISCUSSION TITLE
Use of the files which are generated in the download folder by the automatic Bisq updater

### DISCUSSION THREAD

**Discussion by RequestPrivacy** [bisq-network/bisq#5545] [Q&A]
If I download & verify a new bisq version manually, my current understanding is that I need 3 files: 

- The signing key (once downloaded, the key can be used until he is revoked)
- The new bisq release
- The .asc signature file of the release, to verify its integrity and as proof that the release has been done by the signing key

The automatic bisq updater generates a multitude of other files which type I recon but I'm pretty unsure why they are needed additionally (see picture). Can someone explain to me why there are more than 2 or 3 files? Are they really necessary? It is confusing (at least for me) and clutters the download folder. Maybe it is even possible to remove all files after updating automatically so that no files "remain" in the download folder?!

![Download_folder](https://user-images.githubusercontent.com/72392544/120218989-0ef0a900-c22a-11eb-9040-bf48c14dbc65.png)

**Comment by Conza88** [comment]
I would also be interested in best practice new installation methods.. how to.
